src/composables/useMessageInfo.js

```diff
@@ -17,19 +17,13 @@ const canEditMessage = getCapabilities()?.spreed?.features?.includes('edit-messa
 /**
  * Check whether the user can edit the message or not
  *
- * @param {import('vue').Ref} tokenRef conversation token
- * @param {import('vue').Ref} messageIdRef message id to edit
- * @return {import('vue').ComputedRef<boolean>}
+ * @param {import('vue').Ref} message message object
+ * @return {Record<string, import('vue').ComputedRef<boolean>>}
  */
-export function useMessageInfo(tokenRef = ref(null), messageIdRef = ref(null)) {
-	const token = unref(tokenRef)
-	const messageId = unref(messageIdRef)
+export function useMessageInfo(message = ref({})) {
```

Comment:
I don't think, the default value, especially an empty object which is not a special case for a message object, makes sense here.

`if (...!message.value)` also doesn’t work for `useMessageInfo()` with `{}` value.

Reply:
> Maybe, check for any key present in object?

We can check for `id` property only.

But in a follow-up (not right now) I'd refactor it and replace this `{}` value with explicit `null`.